### PR TITLE
chore: both users who change either auth or email are shown same logout screen

### PR DESCRIPTION
**Pull request by JaideepGuntupalli**
## What does this PR do?

both users who change either auth or email are shown same logout screen

### CODE REVIEW

**Comment by JaideepGuntupalli** [comment]
hey @emrysal, can you pls review and merge this so i could merge yours into the actual pull request?

File: apps/web/pages/auth/logout.tsx [hunk 1]
```diff
@@ -29,11 +30,17 @@ export function Logout(props: Props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [props.query?.survey]);
   useEffect(() => {
-    if (props.query?.changeEmail === "true") {
-      setChangeEmail(true);
+    if (props.query?.emailChange === "true") {
```
Comment (emrysal) [inline apps/web/pages/auth/logout.tsx:33]: you can actually use the query instead of useState here

**Review by emrysal — APPROVED** [review]
Approved, but we should aim to keep state down.

File: apps/web/pages/auth/logout.tsx (hunk 1 above)
Comment (JaideepGuntupalli) [reply apps/web/pages/auth/logout.tsx:33]: Thanks for the feedback! i thought of the same but i thought why change it and followed the existing pattern. Will keep that in mind.

**Pull request merged**